scripts/createDB.sql

```diff
@@ -0,0 +1,213 @@
+-- Create a local db user in order for the backend to be able to access the data.
+drop user if exists 'admin'@'localhost';
+create user 'admin'@'localhost' identified by 'admin';
+grant all privileges on * . * to 'admin'@'localhost';
+ALTER USER 'admin'@'localhost' IDENTIFIED WITH mysql_native_password BY 'admin';
+flush PRIVILEGES;
+
+drop schema if exists `soen_390_db`;
+
+create schema `soen_390_db`;
+
+USE `soen_390_db`;
+
+CREATE TABLE `soen_390_db`.`user` (
+  `userID` INT NOT NULL AUTO_INCREMENT,
+  `name` VARCHAR(45) NOT NULL,
+  `role` VARCHAR(45) NOT NULL,
+  `email` VARCHAR(45) NOT NULL,
+  `password` CHAR(60) NOT NULL,
+  UNIQUE INDEX `email_UNIQUE` (`email` ASC) VISIBLE,
+  PRIMARY KEY (`userID`))
+
+  collate = utf8mb4_unicode_ci;
+  
+CREATE TABLE `soen_390_db`.`customer_order` (
+  `customer_order_id` INT NOT NULL AUTO_INCREMENT,
+  `status` VARCHAR(45) NOT NULL,
+  `date` DATE NOT NULL,
+  `price` DECIMAL(10,2) NOT NULL,
+  PRIMARY KEY (`customer_order_id`))
+  
+  collate = utf8mb4_unicode_ci;
+  
+CREATE TABLE `soen_390_db`.`production_order` (
+  `production_order_id` INT NOT NULL AUTO_INCREMENT,
+  `status` VARCHAR(45) NOT NULL,
+  `date` DATE NOT NULL,
```

Comment:
It would be interesting to have the start and finish dates for these orders for when viewing stats once the order is done